plugins/frontend/backstage-plugin-iframe/src/components/IFrameComponent.tsx

```diff
@@ -109,7 +152,14 @@ const IFrameFromSrc = (props: IFrameProps) => {
  * @public
  */
 export const IFrameCard = (props: IFrameComponentProps) => {
-  const { src, srcFromAnnotation, height, width, title } = props;
+  const {
+    src,
+    srcFromAnnotation,
+    srcWithAnnotationReplacements,
```

Comment:
Let's call this something like templatedSrc or srcWithTemplate or similar I think

Reply:
Agree! Thanks.

Updated to `templatedSrc`